core/domain/takeout_service_test.py

```diff
@@ -58,6 +59,7 @@ class TakeoutServiceUnitTests(test_utils.GenericTestBase):
     USER_1_EMAIL = 'user1@example.com'
     GENERIC_USERNAME = 'user'
     GENERIC_DATE = datetime.datetime(2019, 5, 20)
+    GENERIC_EPOCH = time.mktime(datetime.datetime(2019, 5, 20).timetuple())
```

Comment:
Here and elsewhere -- just use utils.get_time_in_millisecs() or utils.get_current_time_in_millisecs() perhaps?

Reply:
Done